src/tallies/filter_cell.cpp

```diff
@@ -0,0 +1,99 @@
+#include "openmc/tallies/filter_cell.h"
+
+#include <sstream>
+
+#include "openmc/capi.h"
+#include "openmc/cell.h"
+#include "openmc/error.h"
+#include "openmc/xml_interface.h"
+
+namespace openmc {
+
+void
+CellFilter::from_xml(pugi::xml_node node)
+{
+  // Get cell IDs and convert into indices into the global cells vector
```

Comment:
nit: Should this be: // Get cell IDs and convert indices into the global cells vector